src/main/java/org/opentripplanner/inspector/NoThruTrafficEdgeRenderer.java

```diff
@@ -14,36 +14,43 @@
  */
 public class NoThruTrafficEdgeRenderer implements EdgeVertexRenderer {
 
-    private static final Color THRU_TRAFFIC_COLOR = Color.LIGHT_GRAY;
-
-    private static final Color NO_THRU_TRAFFIC_COLOR = Color.RED;
-
-    private static final Color NO_BICYCLE_THRU_TRAFFIC_COLOR = Color.BLUE;
-
-    private static final Color NO_CAR_THRU_TRAFFIC_COLOR = Color.ORANGE;
-
     public NoThruTrafficEdgeRenderer() {
     }
 
     @Override
     public boolean renderEdge(Edge e, EdgeVisualAttributes attrs) {
         if (e instanceof StreetEdge) {
+            int [][] colors = {
+                { 200, 200, 200 }, // no limitations = light gray
+                { 200, 200, 0},    // no walk thru traffic = yellow
+                { 0, 200, 200 },   // no bike thru = cyan
+                { 0, 200, 0 },     // no walk & bike thru = green
+                { 0, 0, 200},      // no car thru = blue
+                { 200, 100, 0},    // no car & walk thru = orange
+                { 200, 0, 200 },   // no car & bike thru = purple
+                { 200, 0, 0 }      // no for all = red
```

Comment:
I suggest you use an array of Color here. The code become simpler and IntelliJ will show the color in the margin. 

Below you should change the `bits -> colorIndex`.

Reply:
I agree, it is certainly better that way.